mopro-core/gpu_explorations/README.md

```diff
@@ -0,0 +1,65 @@
+# GPU exploration
+
+## Report links
+
+- [Benchmarking TrapdoorTechZprize MSM & arkworks(0.4) MSM](https://hackmd.io/ZCxFpQ8AROyYGTl5GLqAGQ)
+
+## Steps to run the benchmarks
+
+There are currently 2 algorithms for benchmarking:
+
+- `arkworks_pippenger`
+- `trapdoortech_zprize_msm`
+
+### mopro-core tests
+
+To run the benchmarks of the instance size of $2^{16}$ on BLS12_377 curve in `mopro-core`, replace `<algorithm_you_want_to_test>` with the algorithm name listed above.
+
+```bash
+cargo test --release --features gpu-benchmarks --package mopro-core --lib -- middleware::gpu_explorations::<algorithm_you_want_to_test>::tests::test_run_benchmark --exact --nocapture
```

Comment:
Since the trapdoortech_zprize_msm is not compatible with BN254 curve, we commented out for now. Do you think we should remove the unused msm? Such as trapdoor's and halo2's?

Reply:
sorry I just saw the message
if you will use the structure to develop for BN254, you can keep it here
do you expect it will be a lot of changes to switch curves?